tests/factories/h_user.py

```diff
@@ -0,0 +1,10 @@
+import factory
+
+from lms import models
+
+HUser = factory.make_factory(  # pylint:disable=invalid-name
+    models.HUser,
+    authority="lms.hypothes.is",
+    username=factory.Faker("hexify", text="^" * 30),
+    display_name=factory.Faker("name"),
+)
```

Comment:
The `username` field looks like one of the usernames that LMS generates for its h users: 30 random hex chars.